app/code/Ludmila/LSNewClasses/Block/Homework11/Homework_OOP.php

```diff
@@ -0,0 +1,73 @@
+<?php
```

Comment:
This file has incorrect name. Dont use underscore characters. It should be HomeworkOOP.php, the same as class name